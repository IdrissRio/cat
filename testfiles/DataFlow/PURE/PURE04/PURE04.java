public class PURE04 {
  void m() {
    Integer x = 0;
    Integer y = x;
    Integer z = y;
  }
}
