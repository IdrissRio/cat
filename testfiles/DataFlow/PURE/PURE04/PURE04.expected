Variable: x
		Abs(__Epsilon)
Variable: y
		Abs(__Epsilon)
Variable: z
		Abs(__Epsilon)
